ca/ca.go

```diff
@@ -601,3 +565,38 @@ func (ca *CertificateAuthorityImpl) issueCertificateOrPrecertificate(ctx context
 
 	return certDER, nil
 }
+
+func (ca *CertificateAuthorityImpl) generateOCSPAndStoreCertificate(ctx context.Context, regID int64, serialBigInt *big.Int, certDER []byte) (core.Certificate, error) {
+	var ocspResp []byte
+	var err error
+	if features.Enabled(features.GenerateOCSPEarly) {
```

Comment:
While you're touching this stuff could you remove this feature check, I don't think we need it anymore. No need to remove the behind the scenes feature stuff, we can do that in a follow up PR.